web-dev/README.md

```diff
@@ -1,4 +1,24 @@
-# Front/Backend web development
+# Web Development
+Web development is a key skill for many software engineers. Have you ever wondered what actually goes into making a site like Google or GitHub? Web developers are the ones responsible for linking the logic of the website (back-end) to how it's presented to users like us (front-end). There are a lot of key components to web development, so for this guide, I'm hoping to introduce you to some key concepts and helpful resources to make your first website.
 
+## Front-end Development
+The distinction between front-end and back-end is crucial. Front-end code refers to what creates the webpage that users see, and the primary languages used in front-end development are **HTML, CSS, and Javascript (JS)**. HTML, or HyperText Markup Language, is what defines the content on a webpage, while CSS, or Cascading Style Sheets, is what helps format it to make it look nice. Using just these two, you can make some really cool looking sites, but they won't actually be able to interact with running code unless you make use of JavaScript. While there are plenty of frameworks and tools to make front-end development easier, knowing these three is vital to becoming a good web developer. I've linked some helpful tutorials for each of these 3 below. I recommend starting with HTML, and then moving on to CSS and JavaScript when you feel comfortable.
 
+* [HTML 1](https://www.w3schools.com/html/), [HTML 2](https://html.com/), [HTML 3](https://www.tutorialspoint.com/html/index.htm)
```

Comment:
quick fix - can you put all the links in seperate lines and instead of just saying "HTML #" as the name, give a slightly more detailed title (eg: [w3schools HTML tutorial](https://w3schools.com/html) - This site gives a basic introduction into what HTML is and you you can get started with it. However, it lacks....(blah blah blah)